harness/determined/common/experimental/_util.py

```diff
@@ -0,0 +1,18 @@
+import enum
+
+from determined.common.api import bindings
+
+
+class OrderBy(enum.Enum):
+    """
+    Specifies whether a sorted list of objects should be in ascending or
+    descending order.
+    """
+
+    ASCENDING = bindings.v1OrderBy.ASC.value
```

Comment:
I don't know if this is standard or "best practices", but I think this is a nice touch.